build.gradle

```diff
@@ -86,7 +88,7 @@ dependencies {
     implementation "net.jodah:failsafe:2.4.1"
 
     testImplementation "org.bouncycastle:bcprov-jdk15on:1.68"
-    testImplementation "org.mockito:mockito-core:3.7.7"
+    testImplementation "org.mockito:mockito-core:4.8.1"
```

Comment:
We definitely should update to the latest versions of our dependencies, but as its a major bump if there are any issues or new deprecations, we can do that in a separate task

Reply:
True Jim, but without this update our CI will fail for Java 17 :\